meta-intel-edison-bsp/recipes-bsp/u-boot/u-boot-common_2018.05.inc

```diff
@@ -20,26 +20,22 @@ PE = "1"
 # otherwise we use upstream with the following patches
 
 EDISON-PATCHES = "\
-	file://0001-x86-tangier-Make-_CRS-for-BTH0-Serialized-to-avoid-w.patch \
-	file://0002-x86-Fix-reference-to-QEMU-variant-of-write_acpi_tabl.patch \
-	file://0003-x86-zImage-Move-subarch-assignment-out-of-cmd_line-c.patch \
-	file://0004-x86-zImage-Propagate-acpi_rsdp_addr-to-kernel-via-bo.patch \
 	file://0005-HACK-dfu-add-optional-timeout.patch \
 	file://0006-HACK-mmc-sdhci-SDHCI-controllers-also-need-power.patch \
```

Comment:
This one no longer applies cleanly due to context changes in `edison.c`. I'll prepare a patch shortly.

Reply:
Fixed in my commit